src/Hook/TitleMoveComplete/NotifyUsers.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace BlueSpice\EchoConnector\Hook\TitleMoveComplete;
+use BlueSpice\Hook\TitleMoveComplete;
+
+class NotifyUsers extends TitleMoveComplete {
+	protected function doProcess() {
+		if( $this->user->isAllowed( 'bot' ) ) {
+			//return true;
```

Comment:
why does it not return here? in the past this was quite important for edits with maintenance scripts to NOT trigger any notifications